src/xmlrpc.php

```diff
@@ -12,7 +12,7 @@
  */
 define( 'XMLRPC_REQUEST', true );
 
-// Some browser-embedded clients send cookies. We don't want them.
+// Some browser-embedded clients send cookies, but are not needed, so discard them.
```

Comment:
```suggestion
// Discard unneeded cookies sent by some browser-embedded clients.
```
A possible alternative. It changes the original comment more, but might be more direct. What do you think @ironprogrammer?